### PR TITLE
Fix #342 - Adding the support of altering user options

### LINKED ISSUE #342: Valid ALTER USER statement throws validation error "unrecognized alter operation"
### Describe the bug

When writing an `ALTER USER` statement in phpMyAdmin, a validation error is thrown, even though the query executes successfully.

### To Reproduce

Put the following query into the phpMyAdmin SQL query editor.

```
ALTER USER trevor REQUIRE NONE;
```

### Expected behavior

Query should not show any validation errors in the phpMyAdmin SQL editor.

### Screenshots

image
image

### Server configuration

- Operating system: Windows 10 w/ Docker Desktop
- Web server: Apache
- Database version: MySQL 8.0.26
- PHP version: 
- phpMyAdmin version: 5.1.1

### Client configuration

- Browser: Firefox 91
- Operating system: Windows 10

### PR DESCRIPTION
**Pull request by iifawzi**
Hi, this PR should add the support of altering user options, not all the options are included though, because I'm not aware of all of them, I've just introduced the base. 

Fixes #342
Signed-off-by: Fawzi E. Abdulfattah <iifawzie@gmail.com>

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
This looks good, to better understand I will cut out and merge into QA and master your newly added tests

**Review by williamdes — APPROVED** [review]
Awesome !
I really like the diff it makes on master branch out files

File: src/Components/AlterOperation.php [hunk 1]
```diff
@@ -129,6 +129,42 @@ class AlterOperation extends Component
         'CHARACTER SET' => 3,
     );
 
+    /**
+     * All user options.
+     *
+     * @var array
+     */
+    public static $USER_OPTIONS = array(
+        'ATTRIBUTE' => array(
```
Comment (williamdes) [inline src/Components/AlterOperation.php:138]: MySQL 8.0 also has `COMMENT` https://dev.mysql.com/doc/refman/8.0/en/alter-user.html

Comment (williamdes) [reply src/Components/AlterOperation.php:138]: Let me know if you want to add it or should I merge ;)

Comment (iifawzi) [reply src/Components/AlterOperation.php:138]: Sorry for being late, I will add it in minutes and add a test for it too.

Comment (williamdes) [reply src/Components/AlterOperation.php:138]: Oh no worries, I am working on another part of the parser

File: src/Components/AlterOperation.php [hunk 2]
```diff
@@ -129,6 +129,46 @@ class AlterOperation extends Component
...
+            1,
+            'var'
+        ),
+        'COMMENT' => array(
```
Comment (williamdes) [inline src/Components/AlterOperation.php:142]: How do you determine the numbers and the format ?

I am working on https://github.com/phpmyadmin/sql-parser/issues/339 and https://github.com/phpmyadmin/sql-parser/blob/24eed885811a62ea243a4bec9d460fe3928f3d29/src/Statements/CreateStatement.php#L675 seems to not work at all

**Review by williamdes — APPROVED** [review]
🚢

File: src/Components/AlterOperation.php (hunk 1 above)
Comment (iifawzi) [reply src/Components/AlterOperation.php:138]: Done, added, and squashed

File: src/Components/AlterOperation.php (hunk 2 above)
Comment (iifawzi) [reply src/Components/AlterOperation.php:142]: For example; 
``` CHANGE FAWZI NAME```

there's 3 keywords in this query, so each one MUST have a different number.. that's it

Comment (iifawzi) [reply src/Components/AlterOperation.php:142]: I think here we want to skip these parameters, with maybe something like getNextOfType(`typehere`, 'SELECT'), not sure though that this's the issue, I will take a look and share my thoughts here

Comment (iifawzi) [reply src/Components/AlterOperation.php:142]: going to take a look and share my thoughts here

**Pull request merged**